Lombiq.Tests.UI/EmbeddedResourceProvider.cs

```diff
@@ -0,0 +1,18 @@
+using System.IO;
+using System.Text;
+
+namespace Lombiq.Tests.UI
+{
+    internal static class EmbeddedResourceProvider
```

Comment:
Can't `ResourceHelper` from Helpful Libraries be used?

Reply:
Unfortunately, `ResourceHelper` doesn't have such method that is needed.

Reply:
Are you sure that `GetEmbeddedFileAsync<T>()` doesn't do the same in the end?

Reply:
Absolutely. Just check that method and its generic and regular argument.